src/lib/models/deployment.js

```diff
@@ -66,6 +73,29 @@ module.exports = class Deployment {
     })
   }
 
+  volumeMounts() {
+    return _.map(_.get(this.spec, 'volumes', []), volume => {
+      const out = {
+        name: volume.name,
+        mountPath: volume.destination,
+      }
+      return out
+    })
+  }
+
+  volumes() {
+
+    return _.map(_.get(this.spec, 'volumes', []), volume => {
+      const out = {
```

Comment:
Same here, for just returning the object.

Reply:
Fixed as well.